src/javascript/app_2/App/Components/Elements/DataTable/data-table.jsx

```diff
@@ -28,21 +28,27 @@ class DataTable extends React.PureComponent {
             children,
             columns,
             footer,
-            getRowLink,
+            getRowAction,
             is_empty,
             onScroll,
         } = this.props;
 
         const TableData =
             <React.Fragment>
-                {this.props.data_source.map((row_obj, id) =>
-                    <TableRow
-                        className={this.props.className}
-                        row_obj={row_obj}
-                        columns={columns}
-                        key={id}
-                        to={getRowLink && getRowLink(row_obj)}
-                    />
+                {this.props.data_source.map((row_obj, id) => {
+                    const action = getRowAction && getRowAction(row_obj);
```

Comment:
both getRowAction are function ?

Reply:
@cakasuma yes, one is checking for its existence and one is the `return value` of the function